<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFilePush</class>
 <widget class="QWidget" name="frmFilePush">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QWidget" name="widgetRight" native="true">
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>6</number>
            </property>
            <item row="0" column="1">
             <widget class="QComboBox" name="cboxListenIP">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labPlayMode">
              <property name="text">
               <string>模式</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="cboxPlayMode">
              <item>
               <property name="text">
                <string>直接播放</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>下载播放</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="cboxFlagType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text">
                <string>标识递增</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>指定标识</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>随机标识</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labListenPort">
              <property name="text">
               <string>端口</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="labMaxCount">
              <property name="text">
               <string>请求</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="cboxMaxCount">
              <property name="editable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtListenPort"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="txtListenIP">
              <property name="text">
               <string>网卡</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="cboxCopyNumber"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="labCopyNumber">
              <property name="text">
               <string>复制</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labFlagType">
              <property name="text">
               <string>策略</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="labPushFlag">
              <property name="text">
               <string>标识</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="txtPushFlag"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QPushButton" name="btnStart">
             <property name="text">
              <string>启动服务</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="btnPreview">
             <property name="text">
              <string>网页预览</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btnRemove">
             <property name="text">
              <string>移除选中</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="btnClear">
             <property name="text">
              <string>清空所有</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="btnAddFile">
             <property name="text">
              <string>添加文件</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="btnAddPath">
             <property name="text">
              <string>添加目录</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>当前在线请求总数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="digitCount">
         <number>6</number>
        </property>
        <property name="intValue" stdset="0">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>不同地址请求总数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberAll">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="digitCount">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="txtHttpUrl">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckDebugVisible">
        <property name="text">
         <string>打印输出窗口</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextEdit" name="txtDebug">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboxListenIP</tabstop>
  <tabstop>txtListenPort</tabstop>
  <tabstop>cboxFlagType</tabstop>
  <tabstop>txtPushFlag</tabstop>
  <tabstop>cboxPlayMode</tabstop>
  <tabstop>cboxCopyNumber</tabstop>
  <tabstop>cboxMaxCount</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnPreview</tabstop>
  <tabstop>btnRemove</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>btnAddFile</tabstop>
  <tabstop>btnAddPath</tabstop>
  <tabstop>txtHttpUrl</tabstop>
  <tabstop>ckDebugVisible</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>txtDebug</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
